models/aether_lat-lon/aether_to_dart.f90

```diff
@@ -0,0 +1,472 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+!
+
+program aether_to_dart
```

Comment:
If I run aether_to_dart with an argument that is larger than the ensemble size, it fails. Yet, it creates a filter_input netcdf file including only dimension variables. I think it should fail before creating any file

Reply:
Good catch!  I'll try to fix that.

Reply:
The fix is to read the model time from the actual member instead of member 0.  
It will fail before the filter_input.nc file is created.